### DISCUSSION TITLE
Welcome to OverAlt Discussions!

### DISCUSSION THREAD

**Discussion by venkatesh-reddi** [venkatesh-reddi/OverAlt#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by venkatesh-reddi** [comment]
Completed all the packages. I'll try adding database for the project tonight.

**Comment by mounika-kota** [comment]
https://gamma.app/docs/OverAlt-v2svxujh5n6f3ut

link for the ppt , please check it out and let me know if we need changes

**Comment by Sakina-n72** [comment]
![Screenshot 2024-09-04 124553](https://github.com/user-attachments/assets/36724c4d-101d-4804-afe5-3534c88780d3)
